src/app/team/documents/proofs/Edit.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Component, createRef } from 'react';
+
+import Header from '../../../../components/header';
+import Button from '../../../../components/button';
+import Selector from '../../../../components/selector';
+import Input from '../../../../components/input';
+
+const DEFAULT_PROOFS = [
+  'None',
+  'Photo ID',
+  'Proof of address',
+  'Proof of income',
+  'Birth certificates',
+];
+
+class ProofsRequiredEdit extends Component {
+  constructor(props) {
+    super(props);
+
+    let proofs;
```

Comment:
Doesn't this need to be initialized to `[]`?
If `this.props.value` is falsey, `proofs` will be undefined, and then `...proofs` will throw a TypeError.